### DISCUSSION TITLE
Editor used by `review`

### DISCUSSION THREAD

**Discussion by KaiserKarel** [crev-dev/cargo-crev#456] [Q&A] [answered]
Calling `cargo crev crate review {CRATE}` opens some editor which I have not configured as the default (actually I'm not sure what it opens, but I cannot edit nor save the template). How can I change this to vscode or nano?

**Comment by dpc** [comment, marked as answer]
Set `EDITOR` environment variable

The code controlling it:
https://github.com/crev-dev/cargo-crev/blob/74563fd3294bf410bd658bea2ed41011074f9e5b/cargo-crev/src/edit.rs#L16

https://askubuntu.com/questions/432524/how-do-i-find-and-set-my-editor-environment-variable

I hope this helps.

**Reply by KaiserKarel** [reply]
Thanks!
